cluster/aws/util.sh

```diff
@@ -411,9 +411,9 @@ function create-security-group {
 
   local sgid=$(get_security_group_id "${name}")
   if [[ -z "$sgid" ]]; then
-	  echo "Creating security group ${name}."
-	  sgid=$($AWS_CMD create-security-group --group-name "${name}" --description "${description}" --vpc-id "${VPC_ID}" --query GroupId)
-	  add-tag $sgid KubernetesCluster ${CLUSTER_ID}
+          echo "Creating security group ${name}."
+          sgid=$($AWS_CMD create-security-group --group-name "${name}" --description "${description}" --vpc-id "${VPC_ID}" --query GroupId)
+          add-tag $sgid KubernetesCluster ${CLUSTER_ID}
```

Comment:
Is this just a spacing change?  Maybe remove from the PR, or match the 2-space approach we seem to use in this file?